app/src/main/res/values/strings.xml

```diff
@@ -316,4 +316,12 @@
     <string name="no_safe_selected_error">Please select a Safe to proceed!</string>
     <string name="channel_signing_request_name">Signing Requests</string>
     <string name="channel_signing_request_description">Requests to sign transactions.</string>
+    <string name="add_device">Add device</string>
+    <string name="security_level">Security level</string>
+    <string name="security_info_weak">Only this device needs to sign to confirm transactions. No recovery option if access to one device is lost.</string>
+    <string name="security_info_good">Two devices needed to sign to confirm transactions. No recovery option if access to one device is lost.</string>
```

Comment:
this is incorrect, with `good` recovery is enabled

Reply:
True. Only one confirming device.